default-rails.yml

```diff
@@ -9,20 +9,20 @@ Rails:
 
 AllCops:
   Exclude:
-    - 'db/schema.rb' # this is a generated file, we don't want to be manually corrected it.
-    - 'node_modules/**/*'
+    # schema.rb is a generated file, we don't want to correct it.
+    # TOFIX: Document / Discuss other 2 file-groups in V2
+    - 'db/schema.rb'
     - '**/*/Brewfile'
+    - 'db/migrate/*'
```

Comment:
In Casebook we do actually lint our migrations

Reply:
So basically from the below, I just dried up some code. WDYT with this>? What are the regular rails conventions?

The issue with the rails config file here (As opposed to the regular ruby one, which I've been working on), is most of the stuff is undocumented. So the "why" is often very ambiguous. I'm happy to remove this, but then this would be a V2 release of the gem (Again not against that, just didn't plan on doing v2 so quickly).

Reply:
Yes some of these are a bit specific to casebook and must just have been copied.  It doesn't help we have three directories of specs.

The migrations are code mostly written by hand so should be linted, but they follow a very specific format so the size rules become quite troublesome.  This should be true of any Rails project

Reply:
So would you prefer

a) Me removing all of the db_migrate refactors in this PR, and then doing some more work and making the next cut a V2
b) Just make a mental note and fix it later?
c) Something else?

Reply:
b should be fine, if that is easiest

Reply:
Happy to do that. Will tackle a v2 release once I have some free time and can tag a few reviewers. So is this RtM now then?